nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -344,7 +343,7 @@ in {
     };
 
     enableTpm2 = mkOption {
-      default = true;
+      default = cfg.package.withTpm2Tss;
```

Comment:
This causes a build failure in lazy-options.json, sadly.

Reply:
Shouldn't it just be `mkPackageOption`? (I.e using defaultText for docs)

Reply:
Yea, it's fixed with `defaultText`.